### PR TITLE
Removing all forklifted content from project

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
The following pages have been added to wagtail and no longer need to be
hardcoded or added in with sheer. All relevant CSS and tests have also
been deleted in this PR. This will need a changelog fix.

## Removals

- Removed sheer templates and hardcoded pages that have been forklifted into wagtail

## Testing

- Run `gulp test:acceptance --sauce=false`
- If you have the recent database dump, make sure the links sync up with the new content in wagtail

## Review

- @jimmynotjim 
- @sebworks 
- @anselmbradford 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by sebworks** [comment]
The second time in a week that I've been lucky enough to cry tears of joy.

crying

File: cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html [hunk 1]
```diff
@@ -27,6 +27,7 @@
 ] %}
 
 {% set nav_items = [(
+<<<<<<< ba40e4c8b03945f8bf8ae9578bce0a99b4b1a2f1
```
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html]: Oops. To delete.

**Comment by jimmynotjim** [comment]
This looks good, but I'm getting failures in the acceptance tests for all the footer links

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
@jimmynotjim Just got the database up and fixed the footer tests. Didn't catch it the first time because they fail when a homepage isn't set. Thanks for the catch!

**Comment by jimmynotjim** [comment]
Looks good. I still get three failures for the Bureau Structure page, but @sebworks is already fixing those. :+1:

**Comment by anselmbradford** [comment]
Looks like the following directories can also be removed:
 - `cfgov/unprocessed/js/routes/budget/`
 - `cfgov/unprocessed/js/routes/the-bureau/`
 - `cfgov/unprocessed/js/routes/browse-filterable/`

Also, does this remove all /offices/ and /sub-pages/ path pages? If so, `cfgov/unprocessed/js/routes/offices/` and `cfgov/unprocessed/js/routes/sub-pages/` can be removed too.

Otherwise, I get the 3 failures @jimmynotjim noted. :+1:

**Comment by KimberlyMunoz** [comment]
The Bureau and Browse Filterable pages are sticking around, so those should probably remain unless there's another reason they would be removed. And yep, I'm leaving the Bureau Structure page failures since it's being worked on outside this PR.

I'll look into the rest of the js and removing that. :scissors:

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
Can I get one more :+1: and :eyes:  just for those most recent changes?

**Comment by anselmbradford** [comment]
LGTM :+1:

**Pull request merged**